src/components/authors-list/AuthorsListItem.jsx

```diff
@@ -10,32 +10,41 @@ import CardMedia from '@material-ui/core/CardMedia'
 import Button from '@material-ui/core/Button'
 import { Grid, Typography } from '@material-ui/core'
 
+import AnimationWrapper from '../animation-wrapper/AnimationWrapper'
+
 const AuthorsListItem = ({ id, name, img, description }) => {
+  const [isAnimated, setAnimated] = useState(false)
```

Comment:
```suggestion
const [isAnimated, setAnimated] = useState(false);
useEffect(() => {
  setAnimated(true);
}, []);
...
<AnimationWrapper pose={isAnimated ? 'on' : 'off'}>
   {children}
</AnimationWrapper>
```

should be extracted into separate component.